source/sdk/io/File.ooc

```diff
@@ -25,11 +25,9 @@ File: abstract class {
 	extension: String { get { getExtension() } }
 	nameWithoutExtension: String { get { getNameWithoutExtension() } }
 	pathWithoutExtension: String { get { getPathWithoutExtension() } }
-	children: VectorList<This> {
-		get {
-			getChildren()
-		}
-	}
+	children: VectorList<This> { get {
+		getChildren()
```

Comment:
Yes, but that's missing in 1000 other places in this file. We want to enforce `this`, so I will be adding it in some future PR. For now, this does exactly what it used to do.